ecs-init/engine/engine.go

```diff
@@ -33,6 +36,11 @@ const (
 	terminalSuccessAgentExitCode = 0
 	terminalFailureAgentExitCode = 5
 	upgradeAgentExitCode         = 42
+	serviceStartMinRetryTime     = time.Millisecond * 500
+	serviceStartMaxRetryTime     = time.Second * 15
+	serviceStartRetryJitter      = 0.10
+	serviceStartRetryMultiplier  = 2.0
+	serviceStartMaxRetries       = math.MaxInt64
```

Comment:
Please add a comment on the intent of retrying forever. It clear this is a big number, but its a big number because its symbolic for "forever".

Reply:
sure will do